src/playground/ec/.storybook/preview.js

```diff
@@ -40,7 +40,7 @@ addParameters({
     },
   ],
   controls: { expanded: true },
-  layout: 'fullscreen',
+  layout: 'padded',
```

Comment:
we should also apply this to EU storybook

Reply:
Oups, thank you 👍